src/index.js

```diff
@@ -1,25 +1,19 @@
 import _ from 'lodash';
-import printResponse from './printResponse';
 import printError from './printError';
 import getEpisodes from './getEpisodes.js';
 import renderData from './renderData.js';
+import { renderLoader, removeLoader } from './renderData.js';
 
+async function main() {
+    
+    renderLoader();
+    
+    const response = await getEpisodes().catch(error => error);
+    
+    removeLoader();
+    const episodes = response.data ? response.data.data.episodes : null;
 
-// Mock information
-const episode = {
-  enumeration: "S01E01",
-  title: "Piloto",
-  airDate: "02/12/2013",
-  characters: ["Rick Sanches", "Morty Smith", "Beth Smith", "Jerry Smith", "Summer Smith",
-               "Jessica", "Mr. Goldenfold", "Frank Palincky", "Davin", "Glenn"],
-  totalOfEpisodes: 41,
-  episodesListed: 1
+    episodes ? renderData(episodes) : printError(response);
 }
 
-
-// Get the episodes from the database and prints it on the console
-getEpisodes().then(printResponse).catch(printError);
-
-
-// Render the episode's information
-renderData(episode);
+main();
```

Comment:
I used main as an example, hehe. You can call it whatever you want! No problems with *main*, tho!